### ISSUE TITLE
Wrong game name in secret achievement

### ISSUE THREAD

**Issue by hexaholic** [Q42/0hh1#27]
Just a minor mistake: the description/body text of the secret achievement (I'm not posting the name here for obvious reasons) contains "0h n0" instead of "0h h1".
Looks like a copy & paste error :)

Seen in Google Play Games on Android.

**Comment by TK0805** [comment]
It isn't a mistake. The name of a companion game is 0h n0. It is most likely referring to that.

**Comment by hexaholic** [comment]
I disagree. The achievement has nothing to do with their other game, I guess I'll just post the full text here:

> **How Very Social Of You**
> Thank you for wanting to share 0h n0 with you friends!

What I did to unlock this was to use the share function of the game. So as I'm sharing 0h h1, it should also say 0h h1 in the text. It's probably just a copy and paste error.
